pages/k8s/install-manual.md

```diff
@@ -0,0 +1,290 @@
+| 1.9.x        | [canonical-kubernetes-150](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-150/archive/bundle.yaml?channel=stable) |
+| 1.8.x | [canonical-kubernetes-132](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-132/archive/bundle.yaml?channel=stable) |
+| 1.7.x | [canonical-kubernetes-101](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-101/archive/bundle.yaml?channel=stable) |
+| 1.6.x | [canonical-kubernetes-38](https://api.jujucharms.com/charmstore/v5/~containers/bundle/canonical-kubernetes-38/archive/bundle.yaml?channel=stable) |
+
+<div class="p-notification--caution">
+  <p markdown="1" class="p-notification__response">
+    <span class="p-notification__status">Note:</span>
+Only the latest three versions of CDK are supported at any time.
+  </p>
+</div>
+
+<a id="additional_configuration" />
+### Additional configuration
+
+To allow Kubernetes to access resources and functionality of the underlying
+cloud upon which it is deployed, additional integrator charms are available. When
+installing with **conjure-up**, these charms are automatically added to the
+deployment and configured accordingly. This table explains which charms are
+used:
+
+| Cloud     | Integrator charm   |  Juju deploy command  | Notes/docs  |
+|------------|----------------------|-------------------------------------------------|-----------------------------------------------------------|
+| AWS        | aws-integrator       | juju deploy cs:~containers/aws-integrator       | [docs](https://jujucharms.com/u/containers/aws-integrator/)       |
+| Azure      | azure-integrator     | juju deploy cs:~containers/azure-integrator     | [docs](https://jujucharms.com/u/containers/azure-integrator/)     |
+| Google     | gcp-integrator       | juju deploy cs:~containers/gcp-integrator       | [docs](https://jujucharms.com/u/containers/gcp-integrator/)       |
+| Local      | N/A                  | N/A                                             |                                                           |
+| OpenStack  | openstack-integrator | juju deploy cs:~containers/openstack-integrator | [docs](https://jujucharms.com/u/containers/openstack-integrator/) |
+| Rackspace  | openstack-integrator | juju deploy cs:~containers/openstack-integrator | [docs](https://jujucharms.com/u/containers/openstack-integrator/) |
+| vSphere    | vsphere-integrator   | juju deploy cs:~containers/vsphere-integrator   | [docs](https://jujucharms.com/u/containers/vsphere-integrator/ )  |
+
+The charm should be deployed and relationships established with both the
+`kubernetes-worker` and `kubernetes-master` charms. For example, in the case of
+AWS:
+
+```bash
+juju deploy cs:~containers/aws-integrator
+juju trust aws-integrator
+juju add-relation aws-integrator kubernetes-master
+juju add-relation aws-integrator kubernetes-worker
```

Comment:
I would really like to see us avoid recommending manually adding and relating the integrator charms if at all possible, because it leads almost inevitably to user mistakes.  I also recall seeing on some clouds (GCP, IIRC, but I'm not sure) an issue where adding the integrator after the cluster had come all the way up didn't properly get the cluster to a state where it could properly interact with the cloud, leading to breakage.  I don't know if that was a one-off issue or whether it was due to something inherent to the clouds, but I haven't had a chance to go back and follow up on trying to reproduce it and debug it.

Either way, using a bundle overlay is always the safer bet.

It might also be useful to link from here to this tutorial: https://docs.jujucharms.com/2.5/en/tutorial-k8s-aws

Reply:
Thats a valid point. However, I feel the YAML approach also abstracts a bit far for people unfamiliar with Juju. I'll have a go at rewording this though and more assuredly recommending using the overlay.

Reply:
@johnsca
Okay, see lines 135-139 above.

As for linking to the Juju docs - we are adding or own integrator pages, so I would prefer to link to them rather than trust the increasingly shifting sands of Juju docs

Reply:
+1 great.  Thanks.